cmd/zdx/README.md

```diff
@@ -0,0 +1,10 @@
+# `zdx`
+
+zdx is command-line utility for debugging and interrogating ".zdx" ZNG files
+ that are used as internal indices for [zar](../zar/README.md) archives.
```

Comment:
```suggestion
that are used as internal indexes for [zar](../zar/README.md) archives.
```
The pcap page refers to indexes rather than indices. I don't prefer one over other, but we should be consistent.